pybaseball/team_results.py

```diff
@@ -95,8 +95,25 @@ def make_numeric(data):
 
 def schedule_and_record(season=None, team=None):
     # retrieve html from baseball reference
+    # sanatize input
+    team = team.upper()
+    df = teams()
+    seasons = df['yearID']
+    team_names = df['teamIDBR']
+    first_occurrences = team_names.drop_duplicates(keep='first')
+    first_occurrences_index = first_occurrences.index.values
+    first_years = pd.DataFrame(seasons[first_occurrences_index])
+    first_years['teamID'] = first_occurrences
+    first_season_map = dict(zip(first_years['teamID'],
+                                first_years['yearID']))
+    if season < first_season_map[team]:
+        m = "Season cannot be before first year of a team's existence"
```

Comment:
If the dict from line 107 is hardcoded, this logic should be modified to allow requests to succeed if a team isn't in the dict (e.g. expansion franchises and team relocations). Otherwise `first_season_map[team]` will throw an error if an unknown `team` is passed